components/Composition/FHeadingWithIcon.jsx

```diff
@@ -8,6 +8,7 @@ import PropTypes from 'prop-types';
 
 export const FHeadingWithIcon = ({
   icon, iconPlacement, iconSize, iconColor, titleColor, titleWeight, titleSize, title, titleAlign, titleStyle, iconStyle,
+  containerStyle, numberOfLines,
```

Comment:
skoro FHeading ma props style to tutaj też trzeba go dorobić nazwałabym go titleStyle i przekazała do wszystkich FHeadingów w tym komponencie do style